source/develop/release-management/features/network/provider-physical-network.md

```diff
@@ -79,6 +79,20 @@ Engine also covers the validation:
 - Physical network must be on the same Data Center as the external network.
 - The custom values (physical network and VLAN ID) must not be specified a when provider physical network is set.
 
+### Import of a provider physical network
+While importing a network with physical network access from an external network
+provider into Engine's database, Engine attempts to map this physical network has to be mapped to the corresponding logical network in Engine.
```

Comment:
Maybe also state what happens if not corresponding network is found here (move from line 93)

Reply:
Done